### PR TITLE
Custom scrollbar in tracks list (fix #344)

### LINKED ISSUE #344: Scrollbar too small
The current scrollbar (I'm testing on Ubuntu) is too thin, such that it's impossible to drag without switching to the native frame. Whenever you go close, the pointer switches to resize instead of drag.

Using the native frame fixes this, but I wanted to raise the question since I imagine it's fairly simple: Can the scrollbar be made longer?

### PR DESCRIPTION
**Pull request by KeitIG**
Fix #344 

- Use [`react-custom-scrollbars`](http://malte-wessel.github.io/react-custom-scrollbars/) in `TracksList` so we can actually scroll with the scrollbar instead of getting in the "resize window" mode
- Remove some CSS hacks to make the alignment of cell looks good when it wasn't

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "queue": "4.2.1",
     "react": "15.6.1",
     "react-bootstrap": "0.31.1",
+    "react-custom-scrollbars": "^4.1.2",
```
Comment (YurySolovyov) [inline package.json]: should we use exact versions without semver-ness modifiers?

File: src/js/components/Shared/TracksList.react.js [hunk 2]
```diff
@@ -121,7 +128,8 @@ export default class TracksList extends Component {
     }
 
     scrollTracksList() {
-        this.setState({ scrollTop : document.querySelector('.tracks-list-body').scrollTop });
+        //console.log(this);
```
[the remark this answers is not in the document]
Comment (KeitIG) [reply src/js/components/Shared/TracksList.react.js]: didn't compromise eslint haha

**Comment by KeitIG** [comment]
Done

**Comment by KeitIG** [comment]
Ty for the review!

**Pull request merged**